opencog/persist/zmq/atomspace/ZMQClient.cc

```diff
@@ -1,11 +1,11 @@
+
 /*
- * opencog/atomspace/ZMQClient.cc
- *
- * Copyright (C) 2008-2010 OpenCog Foundation
```

Comment:
this was oversight on my part, the "more correct" copyright is in `ZMQClient.h`.
The reason why it was this way is because I originally copied structure from your SQL Persistence, then realized that parts of implementation is in ZMQClient, so I moved the contents from Erwin's ZMQClient to my ZMQClient, forgotting to update ZMQClient.cc in the process (but I did update ZMQClient.h).